src/main/java/net/sf/jsqlparser/util/TablesNamesFinder.java

```diff
@@ -366,14 +366,25 @@ public void visit(OverlapsCondition overlapsCondition) {
         overlapsCondition.getRight().accept(this);
     }
 
-    @Override
```

Comment:
I am not going to accept any of those changes because they don't improve readability of the code.